graph_nets/demos_tf2/shortest_path.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
Please make sure this runs in Google Colaboratory, and add the relevant TF2 version to the README (analogous to how it was done for the the sort demo).